documentation/source/installation.rst

```diff
@@ -29,29 +29,29 @@ Otherwise, you can clone with HTTPS:
 Setting up the environment and installing
 -----------------------------------------
 
-``Bluemira`` can be installed into a conda environment using miniforge by running the
+``Bluemira`` can be installed into a conda environment using mambaforge by running the
 following steps in a mac or Ubuntu command terminal.
 
 .. code-block:: bash
 
     # Run the conda installation script
-    # This installs miniforge, if not already present, and sets up a bluemira environment
+    # This installs mambaforge, if not already present, and sets up a bluemira environment
```

Comment:
I think this branch may need a rebase, we're missing the changes from https://github.com/Fusion-Power-Plant-Framework/bluemira/pull/1684. `envsubst` isn't available by default either, so whilst we're here we could add `gettext-base` to the `apt install` step